iml-agent/src/action_plugins/stratagem/action_filesync.rs

```diff
@@ -134,64 +141,177 @@ async fn archive_fids(
     Ok(result)
 }
 
+async fn archive_files(
+    dest_src: &str,
+    lustre_root: PathBuf,
+    file_list: Vec<PathBuf>,
+) -> Result<Vec<FidError>, ImlAgentError> {
+    let result = vec![];
+    tracing::error!(
+        "archive files dest {} lustre {:?} list {:?}",
+        &dest_src,
+        lustre_root,
+        file_list
+    );
+    Ok(result)
```

Comment:
It doesn't look like this fn does anything

Reply:
Currently it does not, just building infrastructure, getting restore in place.